Sources/DolbyIORTSCore/Manager/SubscriptionManager.swift

```diff
@@ -51,22 +50,29 @@ protocol SubscriptionManagerProtocol: AnyObject {
 }
 
 final class SubscriptionManager: SubscriptionManagerProtocol {
-    static let rtsCore = Logger(subsystem: "io.dolby.rtscore", category: String(describing: SubscriptionManager.self))
+    private enum Defaults {
+        static let subscribeURL = "https://director.millicast.com/api/director/subscribe"
+    }
+    private static let logger = Logger(subsystem: "io.dolby.rtscore", category: String(describing: SubscriptionManager.self))
```

Comment:
We properly shall move this out of this class, so it can be used widely in the whole Core module.

Reply:
Because the category is different so as to log the class name also in the logs.

Reply:
However what we have now is a very lighter version of the logger. Can we look at a centralised logger as a separate ticket ?

Reply:
Yep